libs/ktxreader/src/Ktx2Reader.cpp

```diff
@@ -69,9 +69,6 @@ struct FinalFormatInfo {
 //     transcoder_texture_format::cTFPVRTC2_4_RGBA
 //     transcoder_texture_format::cTFPVRTC1_4_RGB
 //     transcoder_texture_format::cTFPVRTC1_4_RGBA
-//     transcoder_texture_format::cTFBC4_R
-//     transcoder_texture_format::cTFBC5_RG
-//     transcoder_texture_format::cTFBC7_RGBA (this format would add size bloat to the transcoder)
```

Comment:
I am somewhat concerned about this comment. Should I not add BC7 to the supported formats here?

Reply:
We should try and see how big the transcoder becomes with extra support for those formats. They are not helpful on mobile nor for WebGL so it can be a concern (our main priority remains mobile and binary size matters a lot there).

Reply:
I quickly checked the size of libktxreader.a on iOS, release build. It increased by 2408 bytes (from 88272). This seems acceptable to me but we are not so much pressured by app size right now, so my definition of 'acceptable' might be different.

I don't know how to check the binary size on WebGL or Android builds so I propose 3 options:
 - The above numbers on the iOS binary size is convincing enough to merge the PR as it is
 - I revert the changes in this file and someone else can add it if needed. We do use Basis Universal compressed textures in .ktx files but we use https://github.com/KhronosGroup/KTX-Software for that
 - I put the changes in this file behind a macro. If so, what is the appropriate macro? `#if defined(__EMSCRIPTEN__) || defined(ANDROID) || defined(IOS)`?